docker/nginx/addons.conf

```diff
@@ -19,6 +19,23 @@ server {
          error_page 404 = @olympia;
     }
 
+    # Route requests to the appropriate locations in the storage volume
+    location /user-media/uploads/ {
+        alias /srv/user-media/shared_storage/uploads/;
+    }
+
+    location /user-media/addons/ {
+        alias /srv/user-media/files/;
+    }
+
+    location /user-media/guarded-addons/ {
+        alias /srv/user-media/guarded-addons/;
+    }
```

Comment:
see https://github.com/mozilla/addons/issues/15025#issuecomment-2358690244

Reply:
I think it will be easier to remove this part and move that to a separate PR.